k6LoadTests.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  duration: '10m',
+  vus: 20,
+  thresholds: {
+    //avg is around 800ms on https://tapahtumat.test.kuva.hel.ninja
+    http_req_duration: ['p(95)<5000'],
+  },
+};
+
+export default () => {
+  const res = http.get('https://tapahtumat.test.kuva.hel.ninja/fi/events');
```

Comment:
This is fixed to staging. Perhaps we could read the value from dot-env + using secrets on workflow. So on master it would be staging, review environment on branch and locally your local server.

Reply:
Just had a discussion with Tomi M. For right now, we keep one hardcoded url (staging). Later we will improve.